scripts/expire-elastic-indices

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import os
+from datetime import date
+
+import httpx
+from sentry_sdk.crons import monitor
+
+
+@monitor(monitor_slug="expire-elasticsearch-indices")
```

Comment:
Il va falloir configurer le monitor côté Sentry si on importe cette commande.

Reply:
yes, ça faisait aussi partie des points dont je voulais parler parce que je suis pas trop au clair sur ce que fait la commande.

Reply:
Les logs du serveur applicatif sont “drainés” (de l’anglais `drain` :wink:) vers ELK (au moins sur le C1). Si ce n’est pas le cas sur IC, on en aura besoin très rapidement, car Clever ne conserve que quelques jours d’historique. Mais les logs s’accumulent dans ELK, on était à 98 % d’espace disque utilisé sur le C1 (donc 88 % pour l’APM :facepalm:). La commande sert à enlever les logs datant de plus d’un an.

Pour le monitor, Sentry a une fonctionnalité de surveillances des crons : https://docs.sentry.io/product/crons/
Le décorateur s’occupe de faire le *check-in* et le *check-out*.

Reply:
Ok, donc la question est de savoir si le monitor sur la commande est utile ou non.
Je pense que c'est pas méga utile, surtout qu'on aura beaucoup moins de logs qu'itou normalement

Reply:
Le monitor est utile pour éviter de risquer de perdre les logs (quand le disque est rempli). C’est normalement quelques clics côté Sentry pour en mettre un en place.

Reply:
Super, merci :heart: